pkgs/tools/misc/screen/default.nix

```diff
@@ -23,6 +23,12 @@ stdenv.mkDerivation rec {
       stripLen = 1;
     });
 
+  postPatch = stdenv.lib.optionalString (stdenv.buildPlatform != stdenv.hostPlatform)
+    # XXX: Awful hack to allow cross-compilation.
+    '' sed -i ./configure \
+           -e 's/^as_fn_error .. \("cannot run test program while cross compiling\)/$as_echo \1/g'
```

Comment:
So it tries to execute `echo` here?

Reply:
Can `autoReconfHook` also solve this?

Reply:
yes, instead of exiting

Reply:
> Can autoReconfHook also solve this?

seems, no
At least simple adding of it into ```nativeBuildInputs``` does not help